src/components/organisms/UiList/stories/ListBox.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <UiList
```

Comment:
Keyboard interaction doesn't work as expected. For example: moving by keyboard from one option to another should be available by key arrow up and arrow down, now is possible using Tab and Tab+shift.

Please check the sections `Keyboard interaction` and `Aria` and add changes to code

Reply:
Please check this section in confluence and fix keyboard navigation:
![image](https://github.com/infermedica/component-library/assets/37744371/a7484660-17d2-4707-96a4-af716f8c7e9c)

Reply:
I left a few more links with details on how to use `ul role="listbox` and an alternative way with `select`:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role
https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-rearrangeable/
I should help to implement the best solution 😍